### ISSUE TITLE
Feature request: disable auto-import

### ISSUE THREAD

**Issue by madneon** [vshaxe/vshaxe#630]
Please give and option to disable auto-import completly.
If not separate settings checkbox than maybe in "**Imports Sort Order**", maybe there, 1 more option to choose from for disabling this functionality?

Inserting some/any code out of current editor's view range is not convenient behaviour for me.

**Comment by AlexHaxe** [comment]
```json
"haxe.codeGeneration": {
    "imports": {
        "enableAutoImports": false
    }
}
```
does that not work for you?

**Comment by madneon** [comment]
@AlexHaxe yes, it works. Thank you for that! Still, it would be nice to have a Setting for that.

**Comment by AlexHaxe** [comment]
well, that is the setting. there is no button in settings UI for it. when you search for `haxe.codeGeneration` in settings UI it points you towards `Edit in settings.json`, so that's how you have to set that option. 
VSCode helps you by providing auto-completion in `settings.json` for all fields and potential values (with defaults) and it shows tooltips.

unfortunately not having such settings in settings UI is a [restriction of VSCode](https://code.visualstudio.com/api/references/contribution-points#contributes.configuration:~:text=Edit%20in%20settings.json) and there is not much one can do to change that. you can only hope that they add support for nested object / array configuration options in settings UI. there is probably an issue / discussion open over at the VSCode repository where you can join in and +1 any request to add it.

**Comment by madneon** [comment]
@AlexHaxe thanks again, I didnt know there is such restriction in VS Code.
